bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/PersistentInbox.java

```diff
@@ -673,29 +674,29 @@ protected void unsetEventPublisher(EventPublisher eventPublisher) {
     }
 
     @Reference
-    protected void setThingTypeRegistry(ThingTypeRegistry thingTypeRegistry) {
+    public void setThingTypeRegistry(ThingTypeRegistry thingTypeRegistry) {
```

Comment:
why is this change necessary (and the other protected to public changes, too)?

Reply:
I think I changed them when Eclipse reported visibility errors in PersistentInboxTest. For some reason they are gone now. I've also moved the InboxOSGITest to another package so it can also use the protected methods.